packages/cli-kit/src/private/node/ui/components/Table/Row.tsx

```diff
@@ -0,0 +1,55 @@
+import ScalarDict from './ScalarDict.js'
+import {Column} from './Column.js'
+import {Box, Text} from 'ink'
+import React from 'react'
+
+interface RowProps<T extends ScalarDict> {
+  line: string
+  rowKey: string
+  data: Partial<T>
+  columns: Column<T>[]
+}
+
+function intersperse<T, TI>(intersperser: (index: number) => TI, elements: T[]): (T | TI)[] {
```

Comment:
It's the pseudo equivalent of `columns.join(separator)`, which isn't feasible in React.
I renamed the function and switched the params order – maybe this might make it clearer?

Reply:
yeh this change helped a lot -- thanks!